column_buffer_reflect.go

```diff
@@ -495,6 +495,10 @@ func writeValueFuncOfMap(columnIndex int16, node Node) (int16, writeValueFunc) {
 	zeroKeyValue := reflect.Zero(keyValueElem)
 
 	return nextColumnIndex, func(columns []ColumnBuffer, levels columnLevels, mapValue reflect.Value) {
+		if mapValue.Kind() == reflect.Pointer {
+			mapValue = mapValue.Elem()
```

Comment:
I believe this would panic if the pointer was `nil`, could we check and handling the same way than nil maps are handled below?

Reply:
Oh nice, I was mistaken!

Thanks for making the update.